docs/sources/enterprise/access-control/usage-scenarios.md

```diff
@@ -34,6 +34,7 @@ Example response:
             "version": 2,
             "uid": "qQui_LCMk",
             "name": "fixed:users:org:edit",
+            "displayName": "users org edit",
             "description": "Allows every read action for user organizations and in addition allows to administer user organizations.",
```

Comment:
```suggestion
            "description": "Allows every read action for user organizations and in addition allows to administer user organizations.",
```
This line does not make sense. Namely, "allows to do what" and "allows who to do what to whom".

Reply:
Would "Enable or disable the ability to ..." work?